pkg/syncer/specsyncer.go

```diff
@@ -151,27 +151,24 @@ func (c *Controller) applyToDownstream(ctx context.Context, gvr schema.GroupVers
 
 	downstreamObj := upstreamObj.DeepCopy()
 	downstreamObj.SetUID("")
+	downstreamObj.SetCreationTimestamp(metav1.Time{})
```

Comment:
No need to set this, it gets set by the api.

Reply:
Removed 👍🏼.